build.gradle

```diff
@@ -205,6 +205,8 @@ dependencies {
   implementation 'com.google.guava:guava'
   implementation 'io.opentelemetry:opentelemetry-api:1.29.0'
   implementation group: 'org.apache.activemq', name: 'artemis-jms-server', version: '2.30.0'
+  implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '10.1.13'
```

Comment:
you can add `ext['tomcat.version'] = '10.1.13'` to the build.gradle outside (below the other 'ext' block) instead

Reply:
resolved